### DISCUSSION TITLE
[FEATURE] Web UI access only from local network

### DISCUSSION THREAD

**Discussion by hdimon** [bunkerity/bunkerweb#284] [Help] [answered]
Hi,

There is rather question than feature request. So if it already could work in this way than maybe it's "documentation request". If it does not work than feature request:).

I've just setup BunkerWeb using config from example https://docs.bunkerweb.io/1.4/web-ui/#installation but replacing bwadm.example.com with mybunker.server. Than I added `192.168.1.2 mybunker.server` on my local PC in hosts file. Everything works fine, when I try to open http://mybunker.server in browser I see BunkerWeb UI, can login and so on. My question is: if I expose this 192.168.1.2:80 in Internet via port forwarding will it be still accessible **only** from local network? Or it will be possible to access admin panel using public IP (I mean including "hack" ways)?

Thank you very much.

**Comment by fl0ppy-d1sk** [comment]
Hello @hdimon,

I think it should work as long as you set the hosts file like `1.2.3.4 mybunker.server` where `1.2.3.4` is your public IP.

**Reply by coding-usa** [reply]
can you give me a video how to install bunkerweb

**Comment by fl0ppy-d1sk** [comment, marked as answer]
And if you want to disable access from outside, maybe you can set `WHITELIST_IP=192.168.0.0/16` and `BLACKLIST_IP=0.0.0.0/0`.

**Reply by itguy327** [reply]
Thank you. I would have simply thought that WHITELIST=192.168.0.0/16 would be sufficient, but maybe not within my realm of understanding

**Reply by SkizzoSaar** [reply]
I have tried to set up like mentioned `WHITELIST_IP=192.168.0.0/16`  and `BLACKLIST_IP=0.0.0.0/0`, except of `1.2.3.4 mybunker.server` is not set in my hosts file but `1.2.3.4` as my public IP has a public resolveable dynamic dns `mybunker.server.tld` (but I don't think this should make a difference to hosts file and public ip address). As result all access (internal as well as external) is forbidden, because in the logs my public facing IP address is logged instead of my internal `192.168.x.x` ip address.

Has anyone an idea why bunkerweb is seeing my public facing IP address instead of the internal one?

**Comment by fl0ppy-d1sk** [comment]
By the way we should have an "allowed IP" feature, see #285 for more info.
